src/test/groovy/graphql/schema/GraphQLDirectiveTest.groovy

```diff
@@ -95,13 +94,12 @@ class GraphQLDirectiveTest extends Specification {
         when:
         def schema = TestUtil.schema(sdl)
         then:
-        schema.getSchemaDirective("d1").name == "d1"
-        schema.getSchemaDirectiveByName().keySet() == ["d1"] as Set
+        schema.getSchemaAppliedDirective("d1").name == "d1"
+        schema.getAllSchemaAppliedDirectivesByName().keySet() == ["d1", "dr"] as Set
```

Comment:
The behaviour slightly changed - the previous method only included non-repeatable directives. The new one includes non-repeatable and repeatable, hence the assert change